src/mod/endpoints/mod_sofia/sofia_glue.c

```diff
@@ -929,10 +929,10 @@ char *sofia_glue_get_extra_headers(switch_channel_t *channel, const char *prefix
 				if ( !exclude_regex || !(/*proceed*/ switch_regex_perform(name, exclude_regex, &re, ovector, sizeof(ovector) / sizeof(ovector[0])))) {
 					const char *hname = name + strlen(prefix);
 					stream.write_function(&stream, "%s: %s\r\n", hname, value);
-					switch_regex_safe_free(re);
 				} else {
 					switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "Ignoring Extra Header [%s] , matches exclude_outgoing_extra_header [%s]\n", name, exclude_regex);
```

Comment:
Would it be more optimal to put into the else?

Reply:
done